virttest/remote.py

```diff
@@ -543,19 +545,23 @@ def scp_to_remote(host, port, username, password, local_path, remote_path,
                            "the interface the neighbour attache")
         host = "%s%%%s" % (host, interface)
 
-    command = ("scp -v -o UserKnownHostsFile=/dev/null "
-               "-o StrictHostKeyChecking=no "
-               "-o PreferredAuthentications=password -r %s "
-               "-P %s %s %s@\[%s\]:%s" %
-               (limit, port, quote_path(local_path), username, host,
-                pipes.quote(remote_path)))
+    command = "scp"
+    if directory:
+        command = "%s -r" % command
+    command += (" -v -o UserKnownHostsFile=/dev/null "
+                "-o StrictHostKeyChecking=no "
+                "-o PreferredAuthentications=password -r %s "
```

Comment:
-l limit
             Limits the used bandwidth, specified in Kbit/s.
it looks need to instead  '-r' with '-l' in this line.

Reply:
@xutian Thank you for review, I will fix it.